config/default.yml

```diff
@@ -653,6 +653,7 @@ Rails/SquishedSQLHeredocs:
   StyleGuide: 'https://rails.rubystyle.guide/#squished-heredocs'
   Enabled: 'pending'
   VersionAdded: '2.8'
+  SafeAutoCorrect: false
```

Comment:
```suggestion
  # Some SQL syntax (e.g. PostgreSQL comments and functions) requires newlines
  # to be preserved in order to work, thus auto-correction is not safe.
  SafeAutoCorrect: false
```

Reply:
@tejasbubane Can you add this one and update `Rails/SquishedSQLHeredocs` cop's doc?

Reply:
Can you add this comment to the cop’s documentation as well? I think it's convenient for users :-)

Reply:
added